client/src/components/relatedProducts/Carousel.css

```diff
@@ -71,6 +71,12 @@
   border: 1px solid #888;
 }
 
+.product-card:hover {
+  background-color: white;
+  opacity: 0.4;
```

Comment:
just CSS stuff to make product cards appear clickable to users before sending to product detail page